lib/debug_utils.py

```diff
@@ -0,0 +1,90 @@
+# Authors: see git history
```

Comment:
Any reason not to put these functions in `debug.py`?  I don't feel that file is too long, and the distinction between what would go in debug_utils.py vs debug.py isn't clear to me.

Reply:
Only reason was that I needed the inkex library not to be imported yet. Not before sys.path is modified.

Reply:
Oh, yup, that makes sense.  How about if we switch to local imports in the functions that need `inkex`?  Then we can have everything in `debug.py`.

Reply:
I considered a local import, but changing the behavior simply by moving the import position seemed too risky to me. What if someone moves it elsewhere over time, and suddenly the program's behavior changes? Plus, there are other indirect imports that import inkex.